index.html

```diff
@@ -12,6 +12,7 @@
       referrerpolicy="no-referrer"
     />
     <title>My Portfolio</title>
```

Comment:
- [ ] Please change the webpage title with something more meaningful, You should have your name or something that makes this title unique for you Like `"Mugiira Portfolio"` and it will also help you with SEO.